src/components/AvatarStack/README.md

```diff
@@ -22,7 +22,8 @@ _Link to the Avatar Stack example and demo._
 
 ## Resources
 
-_Links to relevant pages. See www.ably.com/topic/avatar-stack. We link to a lot of resources there. Method links._
+- Read further about what is possible with an [avatar stack](https://ably.com/examples/avatar-stack?utm_source=ably-labs&utm_medium=github&utm_campaign=avatar-stack).
```

Comment:
Can we use 'what is possible' in just one of these sentences & reword the other one @ginotoro

Reply:
Updated

Reply:
Made another suggestion! :)